### DISCUSSION TITLE
Show field content in translation list

### DISCUSSION THREAD

**Discussion by Babelfisch** [directus/directus#5822] [Archive]
Am I doing it wrong or is it not possible to display the content of fields in a translation list?

The default behavior is like this:

<img width="822" alt="Bildschirmfoto 2021-05-22 um 12 47 53" src="https://user-images.githubusercontent.com/1218405/119223861-0f879200-bafc-11eb-945d-153b1e88b7ec.png">

With an empty interface line it shows only the language code:

<img width="817" alt="Bildschirmfoto 2021-05-22 um 12 41 39" src="https://user-images.githubusercontent.com/1218405/119223894-3a71e600-bafc-11eb-8d85-2ec37a607ad3.png">

Now I add `language.name` and my `title` field (String(255)) to the interface and a dummy text:

<img width="832" alt="Bildschirmfoto 2021-05-22 um 12 51 46" src="https://user-images.githubusercontent.com/1218405/119223964-7dcc5480-bafc-11eb-9e15-a067d3113596.png">

This results in this view:

<img width="823" alt="Bildschirmfoto 2021-05-22 um 12 48 33" src="https://user-images.githubusercontent.com/1218405/119223969-8b81da00-bafc-11eb-8b57-4eaafe05651a.png">

All `title` fields have a content but there is only a gray -- for every field.

What’s wrong?

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/3432
